packages/hardhat-polkadot/package.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "@parity/hardhat-polkadot",
+    "version": "0.0.1",
+    "description": "Polkadot bundle of Hardhat plugins",
+    "repository": "github:paritytech/hardhat-polkadot",
+    "homepage": "https://github.com/paritytech/hardhat-polkadot/tree/main/packages/hardhat-polkadot",
+    "author": "Parity Technologies <admin@parity.io>",
+    "license": "MIT",
+    "main": "dist/src/index.js",
+    "types": "dist/src/index.d.ts",
+    "keywords": [
+        "ethereum",
+        "smart-contracts",
+        "hardhat",
+        "hardhat-plugin",
+        "Polkadot"
+    ],
+    "scripts": {
+        "build": "tsc --build .",
+        "clean": "rimraf dist"
+    },
+    "files": [
+        "dist/src/",
+        "src/",
+        "LICENSE",
+        "README.md"
+    ],
+    "dependencies": {
+        "@parity/hardhat-polkadot-node": "workspace:^",
+        "@parity/hardhat-polkadot-resolc": "workspace:^"
+    },
+    "peerDependencies": {
+        "@parity/hardhat-polkadot-node": "workspace:^",
+        "@parity/hardhat-polkadot-resolc": "workspace:^"
+    },
```

```suggestion
    "dependencies": {
        "@parity/hardhat-polkadot-node": "workspace:^",
        "@parity/hardhat-polkadot-resolc": "workspace:^"
    },
    "peerDependencies": {
            "hardhat": "^2.22.18"
    },
```
No need to have them on both, and we should also add `hardhat` because any project would need it. This will be not be used 99% of the time, but it could come in handy.